src/Catty/Store/StoreProjectUploader.swift

```diff
@@ -60,8 +61,8 @@ final class StoreProjectUploader: StoreProjectUploaderProtocol {
         self.fileManager = fileManager
     }
 
-    func upload( project: Project, completion: @escaping (StoreProjectUploaderError?) -> Void, progression: ((Float) -> Void)?) {
-        guard let zipFileData = fileManager.zip(project) else { completion(.zippingError); return }
+    func upload(project: Project, completion: @escaping (StoreProjectUploaderError?, String?) -> Void, progression: ((Float) -> Void)?) {
```

Comment:
Please also incorporate your changes in the tests. Ideally please also change the order of those parameters so that the `projectId` is the first parameter, similar like in the `StoreProjectDownloaderProtocol`.